src/lib/tls/tls_exceptn.h

```diff
@@ -42,7 +42,7 @@ class BOTAN_PUBLIC_API(2,0) Unexpected_Message final : public TLS_Exception
    {
    public:
       explicit Unexpected_Message(const std::string& err) :
-         TLS_Exception(Alert::UNEXPECTED_MESSAGE, err) {}
+         TLS_Exception(AlertType::None, err) {}
```

Comment:
Not sure how this happened, but it broke the ASIO stream test.

```suggestion
         TLS_Exception(AlertType::UnexpectedMessage, err) {}
```

Reply:
I think I edited that one manually instead of via a script